tests/integration/ceph_smoke_test.go

```diff
@@ -94,6 +94,7 @@ func (s *SmokeSuite) SetupSuite() {
 		ConnectionsCompressed:     true,
 		UseCrashPruner:            true,
 		EnableVolumeReplication:   true,
+		EnableCSINFS:              true,
```

Comment:
This is the only location where this value is set, and it is set to `true`. Do we need this to be configurable? @travisn did you want the ability to disable NFS?